### PR TITLE
Fix note about ? wildcard in EnumerateFiles()

### PR DESCRIPTION
**Pull request by gewarren**
In response to https://github.com/dotnet/dotnet-api-docs/issues/4052#issuecomment-1009379708.

This logic also applies to GetFiles(), so I added the include reference there as well.

### CODE REVIEW

**Comment by gewarren** [comment]
Ping @carlossanlop

File: xml/System.IO/Directory.xml [hunk 1]
```diff
@@ -2907,16 +2897,10 @@ An I/O error occurred.</exception>
...
 
-> [!NOTE]
->  When you use the asterisk wildcard character in a `searchPattern` such as "\*.txt", the number of characters in the specified extension affects the search as follows:
-> 
-> -   If the specified extension is exactly three characters long, the method returns files with extensions that begin with the specified extension. For example, "\*.xls" returns both "book.xls" and "book.xlsx".
-> -   In all other cases, the method returns files that exactly match the specified extension. For example, "\*.ai" returns "file.ai" but not "file.aif".
-> 
->  When you use the question mark wildcard character, this method returns only files that match the specified file extension. For example, given two files, "file1.txt" and "file1.txtother", in a directory, a search pattern of "file?.txt" returns just the first file, whereas a search pattern of "file\*.txt" returns both files.
+[!INCLUDE[enumerate-files-net-framework](~/includes/enumerate-files-net-framework.md)]
```
Comment (carlossanlop) [inline xml/System.IO/Directory.xml:2900]: Nice reuse. Let's save some disk space :)

File: includes/enumerate-files-net-framework.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
...
+> **.NET Framework only:** When you use the asterisk wildcard character in `searchPattern` and you specify a three-character file extension, for example, "\*.txt", this method also returns files with extensions that *begin* with the specified extension. For example, the search pattern "\*.xls" returns both "book.xls" and "book.xlsx". This behavior only occurs if an asterisk is used in the search pattern and the file extension provided is exactly three characters. If you use the question mark wildcard character somewhere in the search pattern, this method returns only files that match the specified file extension exactly. The following table depicts this anomaly in .NET Framework.
```
Comment (carlossanlop) [inline includes/enumerate-files-net-framework.md:2]: FYI @JeremyKuhne and @dotnet/area-system-io

File: includes/enumerate-files-net-framework.md [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
 >
-> | Files in directory  | Search pattern | .NET 5+ returns   | .NET Framework returns  |
-> |---------------------|----------------|-------------------|-------------------------|
-> | file.ai, file.aif   | *.ai           | file.ai           | file.ai                 |
-> | book.xls, book.xlsx | *.xls          | book.xls          | **book.xls, book.xlsx** |
-> | file.ai, file.aif   | ?.ai           | file.ai           | file.ai                 |
-> | book.xls, book.xlsx | ?.xls          | book.xls          | book.xls                |
```
Comment (carlossanlop) [inline includes/enumerate-files-net-framework.md:9]: Good to collapse, these two were basically the same example.

**Review by carlossanlop — APPROVED** [review]
LGTM :shipit:

**Pull request merged**